bin/sandbox

```diff
@@ -1,6 +1,12 @@
 #!/usr/bin/env ruby
 
-require_relative '../lib/mruby_engine'
+require "pathname"
+ENV["BUNDLE_GEMFILE"] ||= File.expand_path("../../Gemfile",
+  Pathname.new(__FILE__).realpath)
```

Comment:
as discussed IRL i've replaced one of the `..` by using `dirname` which make it easier to read